packages/fxa-auth-server/lib/routes/password.js

```diff
@@ -773,51 +772,54 @@ module.exports = function (
         auth: {
           strategy: 'sessionToken',
         },
-        response: {
-          schema: isA
-            .object({
-              authPW: isA.string(),
-            }),
+        validate: {
+          payload: isA.object({
+            authPW: isA.string(),
+          }),
         },
       },
-      handler: async function(request) {
+      handler: async function (request) {
         log.begin('Password.create', request);
         const sessionToken = request.auth.credentials;
         const { uid } = sessionToken;
 
         const { authPW } = request.payload;
-        
+
         const account = await db.account(uid);
         // We don't allow users that have a password set already to create a new password
         // because this process would destroy their original encryption keys and might
         // leave the account in an invalid state.
         if (account.verifierSetAt > 0) {
           throw error.cannotCreatePassword();
         }
-        
+
         // Users that have enabled 2FA must be in a 2FA verified session to create a password.
         const hasTotpToken = await otpUtils.hasTotpToken(account);
-        if (hasTotpToken &&
-          (sessionToken.tokenVerificationId || sessionToken.authenticatorAssuranceLevel <= 1)
-      ) {
+        if (
+          hasTotpToken &&
+          (sessionToken.tokenVerificationId ||
+            sessionToken.authenticatorAssuranceLevel <= 1)
+        ) {
           throw error.unverifiedSession();
         }
-        
+
         const authSalt = await random.hex(32);
-        const password = new Password(
-          authPW,
-          authSalt,
-          config.verifierVersion,
-        );
+        const password = new Password(authPW, authSalt, config.verifierVersion);
         const verifyHash = await password.verifyHash();
 
         // Accounts that don't have a password set, also do not have encryption keys therefore
         // we generate one for them.
         const wrapWrapKb = await random.hex(32);
-        
-        await db.createPassword(uid, authSalt, verifyHash, wrapWrapKb, verifierVersion);
 
-        return {};
+        const passwordCreated = await db.createPassword(
+          uid,
+          authSalt,
+          verifyHash,
+          wrapWrapKb,
+          verifierVersion
+        );
+
+        return passwordCreated;
```

Comment:
This needs to return `{}` or else hapi will fail the request. I tested this locally and was able to create a password.

Reply:
Huh really, do you mean `return { passwordCreated }`? It seemed to work locally for me as-is, but I can easily change it and pull it out of the object [here](https://github.com/mozilla/fxa/pull/12798/files#diff-6b17e5dc6fba78bd8ea0804adbad3b13f1b3dbe23eb56d97ba669beb5dbfc9b3R389) 🤔